src/app/tasks/modals/grade-task-modal/grade-task-modal.component.scss

```diff
@@ -0,0 +1,73 @@
+$primary-blue: #3939ff;
+
+::ng-deep {
+  .mat-accent .mat-slider-thumb-label-text {
+    color: white;
+  }
+
+  .mat-accent .mat-slider-track-fill {
+    background-color: $primary-blue;
+  }
+
+  .mat-accent .mat-slider-thumb {
+    background-color: $primary-blue;
+  }
+
+  .mat-accent .mat-slider-thumb-label {
+    background-color: $primary-blue;
+  }
+
+  .mat-slider-min-value.mat-slider-thumb-label-showing .mat-slider-thumb {
+    background-color: $primary-blue;
+  }
+
+  .mat-slider-min-value.mat-slider-thumb-label-showing.cdk-focused .mat-slider-thumb-label {
+    background-color: $primary-blue;
+  }
+
+  mat-dialog-container.mat-dialog-container {
+    padding: 0;
+  }
+}
+
+.grade-task-modal {
+  width: 450px;
+  padding-left: 2rem;
+  padding-right: 2rem;
+}
+
+.slider {
+  width: 100%;
+  margin-top: 1rem;
```

Comment:
**suggestion:** What do you think about using `padding-top` instead to avoid breaking the ecapsulation?